cmd/crc/cmd/start.go

```diff
@@ -251,18 +258,32 @@ type templateVariables struct {
 }
 
 func writeTemplatedMessage(writer io.Writer, s *startResult) error {
-	parsed, err := template.New("template").Parse(startTemplate)
+	parsed, err := template.New("template").Parse(startTemplateForPodman)
 	if err != nil {
 		return err
 	}
-
+	// This should be replaced with the config preset once
+	// we start using preset as part of config.
+	if s.ClusterConfig.ClusterCACert != "" {
```

Comment:
I think this is added later in the PR? Why wasn't this commit moved after the introduction of presets or at least updated after presets are added?